codespell_lib/data/dictionary.txt

```diff
@@ -8001,11 +8024,13 @@ bnecause->because
 bnndler->bundler
 boads->boards
 boardcast->broadcast
+boardin->boarding, board in,
 boaut->bout, boat, about,
 bobard->board, bombard,
 bocome->become
 boddy->body
 bodiese->bodies
+bodin->boding, bod in,
```

Comment:
Is `bod` a verb in contemporary English?
```suggestion
bodin->boding, bod in, bodkin, boudin,
```

Reply:
I've only ever seen `baud` myself, but potentially.

Reply:
I cannot find such a verb in the [OED](https://www.oed.com/search/dictionary/?q=bod).

Reply:
I mean, I cannot find such a verb `baud`.